### PR TITLE
Feature/472 43 header footer

### PR DESCRIPTION
**Pull request by ntoombs19**

### CODE REVIEW

File: etc/view.xml [hunk 1]
```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
Comment (Brengineer) [inline etc/view.xml]: Don't worry about the year. My Alfred snippet was updated weeks ago. 😉

File: footer.phtml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<div class="container">
```
Comment (erikhansen) [inline footer.phtml]: @ntoombs19 I'm assuming this is temporary content until the static blocks are setup? Can you remove all of this hard-coded content so I can merge this PR?

File: web/css/source/rebar/_background_images.less [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@import 'lib/_background_images.less';
```
Comment (erikhansen) [inline web/css/source/rebar/_background_images.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/_buttons.less [hunk 4]
```diff
@@ -0,0 +1,35 @@
+& when (@media-common = true) {
```
Comment (erikhansen) [inline web/css/source/rebar/_buttons.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/_fluid_typography.less [hunk 5]
```diff
@@ -0,0 +1,36 @@
+@import 'lib/variables/_fluid_typography.less';
```
Comment (erikhansen) [inline web/css/source/rebar/_fluid_typography.less]: I'm concerned about introducing such a global set of styles to this theme…

Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/_grid.less [hunk 6]
```diff
@@ -0,0 +1,108 @@
+@import 'lib/variables/_grid.less';
```
Comment (erikhansen) [inline web/css/source/rebar/_grid.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/_aspect_ratios.less [hunk 7]
```diff
@@ -0,0 +1,59 @@
+@import 'variables/_aspect_ratios.less';
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_aspect_ratios.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/_background_images.less [hunk 8]
```diff
@@ -0,0 +1,28 @@
+/*
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_background_images.less]: Please remove this file

File: web/css/source/rebar/lib/_css_browser_prefixes.less [hunk 9]
```diff
@@ -0,0 +1,26 @@
+//
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_css_browser_prefixes.less]: Please remove this file

File: web/css/source/rebar/lib/_fluid_typography.less [hunk 10]
```diff
@@ -0,0 +1,23 @@
+//
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_fluid_typography.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/_grid.less [hunk 11]
```diff
@@ -0,0 +1,116 @@
+// Grid system
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_grid.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/_responsive_utilities.less [hunk 12]
```diff
@@ -0,0 +1,15 @@
+// Responsive utilities
```
Comment (erikhansen) [inline web/css/source/rebar/lib/_responsive_utilities.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/variables/_aspect_ratios.less [hunk 13]
```diff
@@ -0,0 +1,5 @@
+//
```
Comment (erikhansen) [inline web/css/source/rebar/lib/variables/_aspect_ratios.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/variables/_fluid_typography.less [hunk 14]
```diff
@@ -0,0 +1,25 @@
+//
```
Comment (erikhansen) [inline web/css/source/rebar/lib/variables/_fluid_typography.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/rebar/lib/variables/_grid.less [hunk 15]
```diff
@@ -0,0 +1,9 @@
+//
```
Comment (erikhansen) [inline web/css/source/rebar/lib/variables/_grid.less]: Are the styles in this file being used? I only want to port files from Rebar that are actively being used. Let's keep this theme as minimal as possible and avoid cruft.

File: web/css/source/custom/default/_listings.less [hunk 16]
```diff
@@ -0,0 +1,330 @@
...
+        }
+
+        .filter-options {
+            clear: left;
+        }
+    }
+}
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__s) {
+    .page-products {
+        @media (max-width: 500px) {
```
Comment (Brengineer) [inline web/css/source/custom/default/_listings.less]: @ntoombs19 - Having explained this in person, I'll consider it resolved.

File: web/css/source/custom/default/_header.less [hunk 17]
```diff
@@ -1,30 +1,62 @@
...
+                        &:after {
+                            line-height: 14px;
+                            width: 14px;
+                        }
+                    }
                     &:not(.welcome) {
                         text-transform: uppercase;
                     }
                     &:not(:last-child) {
                         &:after {
```
Comment (Brengineer) [inline web/css/source/custom/default/_header.less]: Well, that was your commit, but I can change it if you like. 😉

File: Magento_Catalog/layout/catalog_product_view.xml [hunk 18]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
Comment (Brengineer) [inline Magento_Catalog/layout/catalog_product_view.xml]: @ntoombs19 This was your commit. You should probably watch out for that.

**Pull request merged**